shotover/src/runner.rs

```diff
@@ -112,13 +118,22 @@ impl Shotover {
         let tracing = TracingState::new(config.main_log_level.as_str(), params.log_format)?;
         let runtime = Shotover::create_runtime(params.stack_size, params.core_threads);
 
+        if params.hotreload {
```

Comment:
Lets remove these logs, we already log that hot reload is enabled in `run_block` which is enough.